app/helpers/channels_helper.rb

```diff
@@ -126,4 +126,43 @@ def time_until_transfer(channel)
       distance_of_time_in_words(Time.now, channel.contesting_channel.contest_timesout_at)
     end
   end
+
+  def get_channel_from_channel_id(channel_id)
```

Comment:
Would you add a comment above function? E.g. inputs and expected output

Reply:
Yeah I agree the helper is not the right place for this.

Reply:
Though I don't think the channel model is the right place for this since it takes a string not a channel.

```
get_channel_from_channel_id(Channel.first.details.channel_identifier)
=> #<Channel id: "55ecd577-0425-420f-8796-78598b06c8a0", publisher_id: "6cdcc152-90f1-4892-8e1a-cf356847594a",..., contest_timesout_at: nil>
```